components/Player/PlayerRecord.jsx

```diff
@@ -5,15 +5,22 @@ import { connect } from 'react-redux';
 import { player } from '../../reducers';
 import styles from './PlayerHeader.css';
 
+//TODO make a dumb component
 export const PlayerRecord = ({ loading, error, wins, losses }) => {
-  const calcWinPercent = (wins, losses) => Math.ceil(10000 * ((wins) / (wins + losses))) / 100;
-
   const getPlayerRecord = () => {
     if (error) return <Error />;
     if (loading) return <Spinner />;
     return (
-      <div className={styles.mmr}>
-        <div>{`${wins} - ${losses} (${calcWinPercent(wins, losses)}%)`}</div>
+      <div>
+        <span className={"text-primary"}>
+          <i className={"fa fa-trophy"}></i>
+          <small>
+            <span className={"text-success"}>{wins}</span>
+            <span> - </span>
+            <span className={"text-danger"}>{losses}</span>
+            <span>{` (${(wins/(wins+losses)*100).toFixed(2)}%)`}</span>
+          </small>
+        </span>
       </div>
```

Comment:
There are some cases we wouldn't want to use connect.

1. We already have the information pulled into the parent (for example the Table component is being connected, but the table row isn't being connected)
2. We are certain we won't use that component anywhere else (where 1. wouldn't apply there).
3. We want to do some sort of presentational component such as PiePercent.

However, we don't pull any of this info into PlayerHeader (it's just accepting a playerId as a prop and rendering these child components with that playerId passed down), so we would have to move that mapStateToProps to there anyways, and we would have to pull in all the information individually because passing down the entire player object will mess up when components should be re-rendered as well as tightly couple those components to the structure of the player object (they should be dumb and not care about the state structure). If we are moving all mapStateToProps to the PlayerHeader, we aren't reducing boilerplate (unless you mean the very minimal 3 or so lines of declaring const mapStateToProps and the connect wrapping), and we are tightly coupling everything together. 

One of the major benefits of react-redux is that we can avoid passing lots of props down to children (or through several layers of children) which is much more confusing (having to trace through several components where things came from).